src/WebJobs.Script.WebHost/Management/HostJsonManager.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management
+{
+    public class HostJsonManager : IHostJsonManager
```

Comment:
We don't need IHostJsonManager, IHostJsonConfigProvider interfaces nor a HostJsonManager Type. Just a single ScriptHost level HostOptionsProvider service that the new API uses to get the resolved options.

Reply:
Make sense! Done